tests/optional_scalars/ScalarStuff.cs

```diff
@@ -21,41 +21,77 @@ public struct ScalarStuff : IFlatbufferObject
   public ScalarStuff __assign(int _i, ByteBuffer _bb) { __init(_i, _bb); return this; }
 
   public sbyte JustI8 { get { int o = __p.__offset(4); return o != 0 ? __p.bb.GetSbyte(o + __p.bb_pos) : (sbyte)0; } }
+  public bool MutateJustI8(sbyte just_i8) { int o = __p.__offset(4); if (o != 0) { __p.bb.PutSbyte(o + __p.bb_pos, just_i8); return true; } else { return false; } }
   public sbyte? MaybeI8 { get { int o = __p.__offset(6); return o != 0 ? __p.bb.GetSbyte(o + __p.bb_pos) : (sbyte?)null; } }
+  public bool MutateMaybeI8(sbyte maybe_i8) { int o = __p.__offset(6); if (o != 0) { __p.bb.PutSbyte(o + __p.bb_pos, maybe_i8); return true; } else { return false; } }
```

Comment:
Shouldn't these generated output use nullable types? I think we need to update the codegen logic to handle this.